### ISSUE TITLE
Question abou main.py testset

### ISSUE THREAD

**Issue by lbg030** [johnsk95/PT4AL#8]
Hi, thank you so much for your sharing your work.

main.py 44 line
`testset = Loader2(is_train=False,  transform=transform_test)`

I think it works trainset, so should I fix it like this?
`testset = Loader(is_train=False,  transform=transform_test)`

**Comment by seominseok0429** [comment]
yes. i will fix it. (It was a long time ago, so I'll check it out and let you know the results.)
